app/post.go

```diff
@@ -393,6 +393,10 @@ func (a *App) handlePostEvents(post *model.Post, user *model.User, channel *mode
 		return err
 	}
 
+	a.Srv.Go(func() {
+		_, _ = a.SendAutoResponseIfNecessary(channel, user)
```

Comment:
You don't need to use blank identifiers if all return values are being discarded.

Reply:
Ha, Joram commented on the same thing on my other PR. I did that so that anyone reading the code would know that it was being done intentionally, but clearly it's just making it more confusing to do that. I'll fix that